bed/info.go

```diff
@@ -61,3 +61,11 @@ func IsSelfOverlapping(b []Bed, verbose int) bool {
 	}
 	return false
 }
+
+// OverlapSize will return an int corresponding to the overlap size between two Bed elements. If the elements are non-overlapping it will return 0.
```

Comment:
I just found that there's a function called OverlapLength in bed/compare.go. Should these 2 be merged?

Reply:
Yeah I think @RavenYL is right, if you if you look at [vertgenlab/gonomics/bed/compare.go](https://github.com/vertgenlab/gonomics/blob/main/bed/compare.go#L105-L113), something similar has been implemented. I am in favor of updating `OverlapLength` to:

`return numbers.Min(a.ChromEnd, b.ChromEnd) - numbers.Max(a.ChromStart, b.ChromStart)`

Reply:
lol i tried to look to see if this function already existed aparently i just missed it. I will delete overlapSize and edit overlapLength to what you suggested Eric